src/Missings.jl

```diff
@@ -206,15 +206,19 @@ else
     Base.float(A::AbstractArray{Missing}) = A
 end
 
+if isdefined(Base, :adjoint)
+Base.adjoint(d::Missing) = missing
```

Comment:
This should be defined in Base as well, right?

Reply:
Yeah, probably.

Reply:
https://github.com/JuliaLang/julia/pull/25502

Reply:
Also add indentation?